content/en/downloads/_index.md

```diff
@@ -9,7 +9,7 @@ images: []
 weight: 100
 layout: default
 ---
-The narayana-full zip is not downloadable anymore from the website, but you can still download single components from [maven central](https://mvnrepository.com/artifact/org.jboss.narayana), build it from github (i.e. [7.2.2.Final](https://github.com/jbosstm/narayana/tree/7.2.2.Final)) or download it from github tag (i.e [7.2.2.Final](https://github.com/jbosstm/narayana/releases/tag/7.2.2.Final))
+The narayana-full zip is not downloadable anymore from the website, but you can still download single components from [Maven central](https://mvnrepository.com/artifact/org.jboss.narayana), build it from github (i.e. [7.2.2.Final](https://github.com/jbosstm/narayana/tree/7.2.2.Final)) or download it from github tag (i.e [7.2.2.Final](https://github.com/jbosstm/narayana/releases/tag/7.2.2.Final))
```

Comment:
I think this is actually not part of JBTM-3995. It's fine for me to stay in this commit but I am expecting it is unrelated to GPG. If there was something GPG'y here please do let me know.

Reply:
Yep, it is unrelated to JBTM-3995. I'll try to back it out